CHANGELOG.md

```diff
@@ -28,13 +28,15 @@ This is a part of aligning `headscale`'s behaviour with Tailscale's upstream beh
 - Add API Key support
   - Enable remote control of `headscale` via CLI [docs](docs/remote-cli.md)
   - Enable HTTP API (beta, subject to change)
+- OIDC login will map user to an associated namespace.
```

Comment:
When we have landed on how this work, we should elaborate here, and in the example config

Reply:
Should I remove changelog modifications ?